docs/getting-started/visualize-data.md

```diff
@@ -17,3 +17,7 @@ We offer various chart types to choose from based on different scenarios.
 ![scatter](../public/dashboard-scatter.jpg)
 
 We are committed to the ongoing development and iteration of this open source project, and we plan to expand the application of time series data in monitoring, analysis, and other relevant fields in the future.
+
+## Embedded Dashboard in GreptimeDB
+
+The Dashboard can be embedded into GreptimeDB's binary. Build GreptimeDB with feature `servers/dashboard` enabled from workspace root. For example, `cargo build --feature servers/dashboard`. The building process will automatically download the distribution of the Dashboard. After the building process succeeded, you can start GreptimeDB normally, and visit the HTTP endpoint `dashboard` to use the Dashboard locally. 
```

Comment:
@MichaelScofield I wonder if we can offer additional information to help users, for example:
1. Besides embedding the dashboard by building the source, is it possible to provide a pre-prepared docker image or compiled binary( This would be beneficial as guide under `Getting started` section, where we'd better assume users do not have the `cargo` installed);
2. Is it possible to specific the exact HTTP endpoint, like `http://server_address:port`?